vividus/src/main/java/org/vividus/configuration/ConfigurationResolver.java

```diff
@@ -208,6 +209,48 @@ private static Multimap<String, String> assembleConfiguration(Properties configu
         return configuration;
     }
 
+    private static Map<String, String> resolveConfiguration(Properties configurationProperties,
```

Comment:
won't the code look better if we use POJO to group profiles, envs and suites?

Reply:
I don't think so, take a look at the assembelConfiguraiton method